test/test_appscale_run_instances.py

```diff
@@ -472,19 +494,40 @@ def test_appscale_in_one_node_cloud_deployment_auto_spot_price(self):
     ).and_return()
 
     # and assume that we can copy over our ssh keys fine
-    self.local_state.should_receive('shell').with_args(re.compile('scp .*[r|d]sa'),
-      False, 5).and_return()
-    self.local_state.should_receive('shell').with_args(re.compile('scp .*{0}'
-      .format(self.keyname)), False, 5).and_return()
-
-    self.local_state.should_receive('shell').with_args(re.compile('scp .*{0}'
-      .format(self.keyname)), False, 5).and_return()
-
-    self.local_state.should_receive('shell').with_args('ssh -i /root/.appscale/bookey.key -o LogLevel=quiet -o NumberOfPasswordPrompts=0 -o StrictHostkeyChecking=no -o UserKnownHostsFile=/dev/null root@public1 ', False, 5, stdin='cp /root/appscale/AppController/scripts/appcontroller /etc/init.d/').and_return()
-
-    self.local_state.should_receive('shell').with_args('ssh -i /root/.appscale/boobazblargfoo.key -o LogLevel=quiet -o NumberOfPasswordPrompts=0 -o StrictHostkeyChecking=no -o UserKnownHostsFile=/dev/null root@elastic-ip ', False, 5, stdin='cp /root/appscale/AppController/scripts/appcontroller /etc/init.d/').and_return()
-
-    self.local_state.should_receive('shell').with_args('ssh -i /root/.appscale/boobazblargfoo.key -o LogLevel=quiet -o NumberOfPasswordPrompts=0 -o StrictHostkeyChecking=no -o UserKnownHostsFile=/dev/null root@elastic-ip ', False, 5, stdin='chmod +x /etc/init.d/appcontroller').and_return()
+    self.local_state.should_receive('shell').\
+      with_args(re.compile('scp .*[r|d]sa'), False, 5).and_return()
+    self.local_state.should_receive('shell').\
+      with_args(re.compile('scp .*{0}'.format(self.keyname)), False, 5).\
+      and_return()
+
+    self.local_state.should_receive('shell').\
```

Comment:
This seems to be the same mock as the line before this. Can you please remove the duplicate line?